lcu_driver/connector.py

```diff
@@ -77,6 +77,9 @@ async def stop(self) -> None:
         :rtype: None
         """
         self._repeat_flag = False
+        if self.connection is not None:
```

Comment:
`<Connection>._close` already gets called on `<Connector>.init`

Reply:
How is a connector being initiated relevant to closing the connection on stop?

Reply:
The `init` method already calls the `close` so the connector doesn't have to...